astropy/visualization/wcsaxes/coordinate_range.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import print_function, division, absolute_import
+
+import warnings
+
+import numpy as np
+
+from ... import units as u
+
+from . import settings
+
+# Algorithm inspired by PGSBOX from WCSLIB by M. Calabretta
+
+
+def wrap_180(values):
```

Comment:
Good idea - I added a TODO for post-merging at the top of this PR, and will open a new issue if this PR is merged.